docs/modules/ROOT/pages/erc1155-metadata-uri.adoc

```diff
@@ -0,0 +1,75 @@
+= ERC-1155 Metadata URI
+
+The OpenZeppelin xref:erc1155.adoc[ERC-1155] Metadata URI extension is needed to manage and store URIs for individual tokens. This extension allows each token to have its own unique URI,
+which can point to metadata about the token, such as images, descriptions, and other attributes.
+This is particularly useful for non-fungible tokens (NFTs) where each token is unique and may have different metadata.
+
+[[usage]]
+== Usage
+
+In order to make an xref:erc1155.adoc[ERC-1155] token with https://docs.rs/openzeppelin-stylus/0.2.0-alpha/openzeppelin_stylus/token/erc1155/extensions/metadata_uri/index.html[Metadata URI] flavour,
```

Comment:
Wrong version in the link, may not work in CI jobs.

Reply:
Should I omit the URL altogether for now?

Reply:
Try with `0.2.0-alpha.2` (this version will include Metadata URI extension) and check if CI passes.